Source/WTF/wtf/PlatformHave.h

```diff
@@ -532,14 +534,10 @@
 #define HAVE_DEVICE_MANAGEMENT 1
 #endif
 
-#if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 120000) || PLATFORM(IOS_FAMILY)
+#if PLATFORM(MAC) || PLATFORM(IOS_FAMILY)
 #define HAVE_NSURLSESSION_WEBSOCKET 1
 #endif
 
-#if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 120000) || PLATFORM(IOS_FAMILY)
```

Comment:
I think I should put this back and do the accompanying change in a follow-up patch.